unfoldingWord/translationCore
# semicolon linting

#### This pull request addresses:

Enforces semicolon linting and fixes preexisting semicolon lint errors.

#### How to test this pull request:

Make sure the tests pass and there are no merge conflicts!


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2750)
